scss/config/_requirements.scss

```diff
@@ -0,0 +1,7 @@
+// External Requirements
+// =====================
+
+@import '../../node_modules/accoutrement-color/sass/color';
+@import '../../node_modules/accoutrement-scale/sass/scale';
+@import '../../node_modules/accoutrement-type/sass/type';
+@import '../../node_modules/accoutrement-layout/sass/layout';
```

Comment:
@mirisuzanne Why are these required now?

Reply:
I don't see this file being used anywhere, and these same modules are imported elsewhere.

Reply:
oh, I know how that happened… the idea was to move them into one place. I'll fix it.